pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -58,7 +58,7 @@ spec:
   # backward compatability. However in the future releases
   # if more and more deployments prefer to use this option,
   # the default can be set to deploy in openebs.
-  - name: DeployInOpenEBSNamespace
```

Comment:
We can't delete this one, just in case there are some storage classes out there that already use this value. 

Instead of introducing a new policy, can we set the value of this policy to default - `true`

Reply:
Done